{"name": "test_network_setup_create_network_invalid_integer_fields_invalid_string_length_and_pattern[configuration-password-&MUKHWgtUUHeUBUPYDsHzjgxxE*GpqVDEV6iTn#qXscg3xqJrf$EPcdn6$2dP45X-422-expected_response_body6]", "status": "passed", "description": "\n        Test network_setup create network failure (invalid integer fields, invalid length or pattern of string fields)\n        ", "attachments": [{"name": "stdout", "source": "085884af-f254-4666-bbbd-e6677988727e-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "key", "value": "'configuration'"}, {"name": "subkey", "value": "'password'"}, {"name": "value", "value": "'&MUKHWgtUUHeUBUPYDsHzjgxxE*GpqVDEV6iTn#qXscg3xqJrf$EPcdn6$2dP45X'"}, {"name": "expected_status_code", "value": "422"}, {"name": "expected_response_body", "value": "{'error': {'statusCode': 422, 'name': 'UnprocessableEntityError', 'message': 'The request body is invalid. See error object `details` property for more info.', 'code': 'VALIDATION_FAILED', 'details': [{'path': '/configuration/password', 'code': 'maxLength', 'message': 'must NOT have more than 63 characters', 'info': {'limit': 63}}]}}"}], "start": 1661841000498, "stop": 1661841000842, "uuid": "bedcb2dc-6fb3-4ef3-88d3-4a203b8ca09f", "historyId": "c5e5e171475350838b596ecf6cf79b7f", "testCaseId": "3ff0c1ed869cb8dfc35387263039eca9", "fullName": "tests.shasta_be.test_network.TestNetwork#test_network_setup_create_network_invalid_integer_fields_invalid_string_length_and_pattern", "labels": [{"name": "tag", "value": "regression"}, {"name": "parentSuite", "value": "tests.shasta_be"}, {"name": "suite", "value": "test_network"}, {"name": "subSuite", "value": "TestNetwork"}, {"name": "host", "value": "GURM-CGW1JG3"}, {"name": "thread", "value": "795-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.shasta_be.test_network"}]}